src/api/nodeRpcCall.js

```diff
@@ -1355,6 +1355,32 @@ const getStakingLedger = async (controller) => {
   return await api.query.staking.ledger(controller);
 }
 
+const bondAndValidate = async (bondValue, walletAddress, callback) => {
+  const api = await getApi();
+  const injector = await web3FromAddress(walletAddress);
+  const bond = api.tx.staking.bond(bondValue, 'Staked');
+  const validate = api.tx.staking.validate({
+    commission: 100000000, // perbill, 10%
```

Comment:
make this configurable please

Reply:
Done:
![image](https://github.com/liberland/liberland_frontend/assets/117277751/ece88488-bb3f-4978-8784-7ce5b0e0bf6e)